fastdds_python/test/api/test_datareader.py

```diff
@@ -743,67 +743,34 @@ def test_get_set_listener(datareader):
     assert(fastdds.StatusMask.all() ==
            datareader.get_status_mask())
 
-    def test(status_mask_1, status_mask_2):
+    def test(status_mask_1):
```

Comment:
I think that there is no need to keep the `_1` suffix anymore.
If the suggestion is accepted, please apply it to all changes/files.
```suggestion
    def test(status_mask):
```

Reply:
Done in ddb9b93